docs/deployer/container.md

```diff
@@ -44,14 +55,19 @@ spec:
 
 ### Contract
 
-In order for the container deployer to interact with the landscaper a contract for imports, exports and the state has to be defined.
+When the image with your program is executed, it gets access to particular information via env variables: 
 
 - The current operation that the image should execute is defined by the env var `OPERATION` which can be `RECONCILE` or `DELETE`.
-- *Imports* can be expected as a json file at the path given by the env var `IMPORTS_PATH`.
+  `RECONCILE` means that the program should just execute its usual installation whereby `DELETE` signals that the
+  corresponding DeployItem was deleted and some optional cleanup could be done.
+- *Imports* are provided as a json file at the path given by the env var `IMPORTS_PATH`.
 - *Exports* should be written to a json or yaml file at the path given by the env var `EXPORTS_PATH`.
-- The optional *state* should be written to the directory given by the env var `STATE_PATH`.
-- The complete state directory will be tarred and managed by the landscaper(:warning: no symlinks)
-- The *Component Descriptor* can be expected as a json file at the path given by the env var `COMPONENT_DESCRIPTOR_PATH`. The json file contains a resolved component descriptor list which means that all transitive component descriptors are included in a list.
+- An optional *state* should be written to the directory given by the env var `STATE_PATH`. The complete state 
+  directory will be tarred and managed by Landscaper(:warning: no symlinks). The last state data are provided 
+  in the next execution or your program. 
+- The *Component Descriptor* can be expected as a json file at the path given by the env var `COMPONENT_DESCRIPTOR_PATH`. 
```

Comment:
we should add that this path is only defined if the provider configuration contains a `componentDescriptor` reference

Reply:
We have changed this but it was not our goal to do a complete rework of this documentation now. We think this document is still not in a good state. It is even unclear what is the target audience. We plan to do a complete rework later.